.github/workflows/build.yml

```
@@ -105,11 +105,69 @@ jobs:
         path: gmenu2x/dist/gmenu2x-miyoo.zip
         if-no-files-found: error # 'error', 'warn', 'ignore'; defaults to `warn`
        
+  # https://github.com/MiyooCFW/buildroot/blob/master/.github/workflows/rootfs.yml
   build-image:
     uses: MiyooCFW/buildroot/.github/workflows/rootfs.yml@master
     with:
-      submodule: buildroot      
-       
+      submodule: buildroot
+
+  # repack final images for beta release with git commit-tag
+  fetch-images:
+    name: MiyooCFW
+    needs: 
+    - build-image
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v3
+    - run: echo "::set-output name=version::$GITHUB_REF_NAME-$(git rev-parse --short HEAD)" | sed 's#/#-#g'
```

A couple of notes here

1) I don't think the `sed` part at the end is doing anything; what's the goal?

2) Github is deprecating this format: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

I think this is the correct replacement, but I'm not 100% sure about the sed part:

```suggestion
    - run: echo "version=$GITHUB_REF_NAME-$(git rev-parse --short HEAD)" | sed 's#/#-#g' >> $GITHUB_OUTPUT
```

---

(It looks like we need to do this in the build-image workflows also.)

---

@nfriedly Currently, the Buildroot repository builds all the Miyoo repositories. But I agree that after the CI run on Buildroot, we could create a Docker image and then use that image for CI on the remaining repositories to quickly check if everything builds correctly.

---

This should resolve a bunch of the other warnings: https://github.com/MiyooCFW/buildroot/pull/26

---

the ``sed`` just takes git repo/branch and  spits repo-branch globally to discard unallowed characters in output's file name. There is  also $(GITHUB_SHA_SHORT) for short HEAD instead in action variables.

Thanks @nfriedly your work and insight in action builds is/was very valuable to us :pray:

Do you have any suggestion about ``with: submodule:`` part?

---

Ok $GITHUB_SHA_SHORT seems not supported currently. I tried with ${GITHUB_SHA::7}, which work for simple output rename but brakes on submodule/kernel end while generating cache key for it, so I'll probably revert that part.

---

@Apaczer 

> the sed just takes git repo/branch and spits repo-branch globally to discard unallowed characters in output's file name. 

Oh, I didn't have the github variable set locally, that's why it didn't do anything when I tried it out :facepalm:

> Do you have any suggestion about with: submodule: part?

Is it causing trouble? If not, I think it's fine to keep using it. I just mentioned that I made it up because you won't find any documentation for it. It's a [generic input](https://docs.github.com/en/actions/using-workflows/reusing-workflows#using-inputs-and-secrets-in-a-reusable-workflow) that I named "submodule".